### ISSUE TITLE
The ROS2 Urdf importer fails to load pxmesh

### ISSUE THREAD

**Issue by michalpelka** [o3de/o3de-extras#230]
Importing a simple URDF test fails to create a collider mesh.
[urdf_test.zip](https://github.com/o3de/o3de-extras/files/11064812/urdf_test.zip)
The assetinfo for dae file is correctly created, the PhysX mesh is generated, but c
MeshColliderComponentRequestsBus complains:
````
<18:13:22> [Error] (AssetDatabase) - No handler was registered for this asset (id={5CB98204-E3EA-55D0-A7E1-9A4F9B7C502F}:2201b8d8,  type={AF3F7D32-1536-422A-89F3-A11E1F5B5A9C})!
````
Here is log:
[Editor.log](https://github.com/o3de/o3de-extras/files/11064808/Editor.log)
o3de-hash: 
 - 952a535aa3616e399cdc22b2b1f05494f60d9cf3  
 - e282c39b6910a7f386179beefc77d7edd9e23c38
 - ee638b1cbb9c885b9a1b8ca172cb9febb471de0a (stabilization/2305)
o3de-extras : stabilization/2305

**Comment by michalpelka** [comment]
It is being fixed in #231.

**Comment by michalpelka** [comment]
Solved with merging #231.
